src/env.ts

```diff
@@ -0,0 +1,23 @@
+import { getCurrent } from "./component/component_node";
```

Comment:
maybe rename the file `hooks.ts` and move `useRef` inside (and later `useEffect`, and possibly others)